sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -675,22 +756,34 @@ export class CertificatesClient {
    * @param certificatePolicy The certificate's policy
    * @param enabled Wether this certificate is enabled or not
    * @param tags Tags for this certificate
+   * @param [options] Optional request parameters
    * @returns Promise<Certificate>
    */
   public async createCertificate(
     name: string,
     certificatePolicy: CertificatePolicy,
     enabled?: boolean,
-    tags?: CertificateTags
+    tags?: CertificateTags,
+    options?: RequestOptions
   ): Promise<Certificate> {
-    let result = await this.client.createCertificate(this.vaultBaseUrl, name, {
-      certificateAttributes: {
-        enabled
-      },
-      tags,
-      certificatePolicy
-    });
-
+    const span = this.createSpan("createCertificate", options);
```

Comment:
Now that this api is changed to take `RequestOptions` and not `RequestOptionsBase`, below is needed as well
```suggestion
    const span = this.createSpan("getCertificateWithPolicy", options && options.requestOptions);
```

Reply:
Thank you! I keep missing these